src/isomorphic/browser/File.ts

```diff
@@ -0,0 +1 @@
+export const File = window.File;
```

Comment:
It absolutely has to be in a separate file with build-time decoupling because of:
* Bundle size for browser (we need to tree-shake the import at the core)
* Importing node stuff into browser causes mayhem :)